src/applications/disability-benefits/2346/config/form.js

```diff
@@ -34,10 +41,12 @@ const formChapters = {
 const formPages = {
```

Comment:
What do you think about `formPageTitlesLookup` or something similar? I think adding `title` to this variable name may make it clearer 🙂

Reply:
After reading more of the code, I'm wondering if we could have something like:
```js
const formPagesLookup = {
  addBatteriesPage: { path: '/some/path', title: 'Add batteries to your order' },
  addAccessoriesPage: { path: '/some/path', title: 'Add accessories to your order' },
  // etc.
}
```